src/utils/sendTaskUpdates.ts

```diff
@@ -5,9 +5,17 @@ export async function sendTaskUpdate(
   completed: string,
   planned: string,
   blockers: string,
+  discordId: string,
+  taskId: string,
   env: env
 ): Promise<void> {
-  const formattedString = `**Completed**: ${completed}\n\n**Planned**: ${planned}\n\n**Blockers**: ${blockers}`;
+  const taskUrl = "https://status.realdevsquad.com/tasks/" + taskId;
```

Comment:
Should I keep this hard-coded here?
It is just to provide the link clickable like of task.

Reply:
Thank you for your clarification

Reply:
added please check